src/functions-reference/mixed_operations.Rmd

```diff
@@ -91,6 +163,14 @@ Convert a one-dimensional array a to a matrix with m rows and n
 columns filled in column-major order.
 `r since("2.15")`
 
+<!-- complex_matrix; to_matrix; (array[] complex a, int m, int n); -->
+\index{{\tt \bfseries to\_matrix }!{\tt (array[] complex a, int m, int n): complex\_matrix}|hyperpage}
+
+`complex_matrix` **`to_matrix`**`(array[] complex a, int m, int n)`<br>\newline
+Convert a one-dimensional array a to a matrix with m rows and n
```

Comment:
[optional] these arguments `a` and `m` and `n` should be single back-tick quoted to make them look like code.